.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on: [push]
+
+jobs:
+  images:
+    name: Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Example image
+        run: ./kas-docker build kas-iot2050-example.yml
```

Comment:
Just curious, why not use kas-container directly?

Reply:
Oops, legacy